### PR TITLE
Add the Item API

### PR DESCRIPTION
**Pull request by RedCommand-dev**

### CODE REVIEW

File: src/main/java/io/github/codeutilities/CodeUtilities.java [hunk 1]
```diff
@@ -76,6 +77,7 @@ public void onInitialize() {
         loader.load(new AutomationFeature());
         loader.load(new Client());
         loader.load(new StreamerModeListeners());
+        new SocketHandler();
```
Comment (BlazeMCworld) [inline src/main/java/io/github/codeutilities/CodeUtilities.java:80]: should probably be loader.load, besides that code should be good even tho i didnt test it

**Review by BlazeMCworld — CHANGES_REQUESTED** [review]

**Comment by RedCommand-dev** [comment]
Changed and tester, works.

**Review by TechStreetDev — DISMISSED** [review]
LGTM

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -1,11 +1,12 @@
 name: Build
+permissions: write-all
```
Comment (DaMan56100) [inline .github/workflows/build.yml:2]: um?

**Review by TechStreetDev — APPROVED** [review]

**Review by BlazeMCworld — APPROVED** [review]

**Pull request merged**